app/src/main/res/layout/activity_add_lodging.xml

```diff
@@ -6,7 +6,48 @@
     android:layout_height="match_parent"
     android:background="#F6CF9A"
 
-    tools:context=".AddLodging">
+    tools:context=".presentation.AddLodging">
+
+    <EditText
+        android:id="@+id/editTextTextPersonName5"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:hint="Location"
+        android:textSize="15dp"
+        app:layout_constraintStart_toStartOf="@id/editTextTextPersonName"
+        app:layout_constraintTop_toBottomOf="@+id/textViewLocation" />
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="88dp"
```

Comment:
the large margin value you're using here suggests you should probably resort to using a Guideline